### ISSUE TITLE
Reproducing LwF experiments

### ISSUE THREAD

**Issue by AntonioCarta** [ContinualAI/continual-learning-baselines#21]
I noticed something strange in my own experiments, related to a change I made a while ago to LwF in the avalanche master branch. Basically, right now the distillation is applied only to the previously active units. Formally, this is the closest solution to the original paper (which only uses multiheads). However, distilling on all the units (as we did previously) results in a better accuracy, probably because of the additional penalization to the new units.

**Comment by AndreaCossu** [comment]
LwF is now working in this repo using the latest Avalanche, I changed the default learning rate to a higher value. So I guess we can stick to the current LwF version since it makes more sense.

**Comment by AntonioCarta** [comment]
Agree. Thanks for the fix.

**Comment by AntonioCarta** [comment]
Did you fix split MNIST? it's not working on my side.

**Comment by AndreaCossu** [comment]
What's the performance? Split MNIST was not failing with LwF ( #18 )

**Comment by AntonioCarta** [comment]
When I created the example it was aroung 73%. See comment in the [original script](https://github.com/ContinualAI/avalanche/blob/master/examples/lwf_mnist.py).

**Comment by AndreaCossu** [comment]
To achieve that performance you need to set different alphas for different experiences and the procedure to produce those alphas is not that well documented. Instead, I took the results from the well-known paper "three scenario for continual learning" so that we can better reference the hyperparameter choice (the accuracy is also aligned with the accuracy usually seen in papers for lwf on split mnist)

**Comment by AntonioCarta** [comment]
ok
